ofl/gowundodum/upstream.yaml

```diff
@@ -0,0 +1,5 @@
+branch: master
```

Comment:
All of your upstream's seem to be missing the `repository_url`

Reply:
That's really strange. I don't know why gftools packager is not adding that line.

Reply:
added